main/conceptual/js-programming/hardened-js.md

```diff
@@ -51,32 +49,32 @@ uses a proposed syntax for [eventual send](./eventual-send.md),
 which we will get to soon.
 :::
 
-## Object-capabilities (ocaps)
+## Object Capabilities (ocaps)
 
 The separation of duties illustrates the core idea
 of _object capabilities_: an object reference familiar
 from object programming _is_ a permission.
 
 In this figure, Alice says: `bob.greet(carol)`
-![alice calls bob.greet(carol)](../../assets/Introduction.svg)
+![alice calls bob.greet(carol)](../assets/Introduction.svg)
 
 If object Bob has no reference to object Carol,
-then Bob cannot invoke Carol; it cannot
+then Bob cannot invoke Carol; Bob can't
 provoke whatever behavior Carol would have.
 
-If Alice has a reference Bob and invokes Bob,
+If Alice has a reference to Bob and invokes Bob,
 passing Carol as an argument, then Alice has both
 used her permission to invoke Bob _and_ given Bob
 permission to invoke Carol.
 
-We refer to these object references as _object-capabilities_ or _ocaps_.
+We refer to these object references as _object capabilities_ or _ocaps_.
 
 ## The Principle of Least Authority (POLA)
 
-OCaps give us a natural way to express the
+Ocaps give us a natural way to express the
```

Comment:
The big C in OCap is on purpose; if you really want to change it, please get review from MarkM.

Reply:
...and here I broke the rule of "leave it alone" that I laid out in the previous comment.  Lovely. I changed it back to "OCaps".